lib/Applify.pm

```diff
@@ -330,7 +359,7 @@ sub _calculate_option_spec {
     $option->{default}
       and ref $option->{default} ne 'ARRAY'
       and die 'Usage option ... default => [Need to be an array ref]';
-    $option->{default} ||= [];
+    $option->{default} = [];
```

Comment:
@jhthorsen I think you are correct, it should be a conditional assignment, but there is a bug as the getopt storage for list options ends up being the `default` array. If the default is an array `$options{$switch}` needs to have a copy of that, not the same reference ([here](https://github.com/jhthorsen/applify/pull/2/commits/9f89084e2b370792844542438484fc9d0cfc8b8c#diff-ae417fcb1a78e526fe1931bf85b03ce7R289)). New tests added in 35ed688 to examine.